plugins/chat/spec/system/admin/csv_exports_spec.rb

```diff
@@ -53,6 +55,7 @@
     assert_message(exported_data.fourth, message_3)
     assert_message(exported_data.fifth, message_4)
   ensure
+    Capybara.save_path = "/tmp/capybara"
```

Comment:
This is brittle as `Capybara.save_path` may change in the future. What we can do here is to store the original value of `Capybara.save_path` in a variable and use the variable to reset it at the end of the test.